ansible/roles/cri/tasks/main.yml

```diff
@@ -1,7 +1,39 @@
 ---
-- name: preflight checks
-  include_tasks: pre_checks.yml
+- name: 'preflight checks'
+  include_tasks: 'pre_checks.yml'
   run_once: true
 
-- name: "setup {{ cri_plugin }} container runtime (cri)"
+- name: 'setup {{ cri_plugin }} as kubernetes container runtime (cri)'
   include_tasks: '{{ cri_plugin }}.yml'
+
+# Ensure the br_netfilter module is loaded and iptables can see bridged traffic
+# https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/#letting-iptables-see-bridged-traffic
+- name: 'ensure br_netfilter module is loaded'
+  community.general.modprobe:
+    name: '{{ item }}'
+    state: 'present'
+  loop:
+  - 'overlay'
+  - 'br_netfilter'
+  tags:
+  - notest
+
+- name: 'enable br_netfilter module loads at system startup'
+  ansible.builtin.copy:
+    dest: '/etc/modules-load.d/kubernetes-br_netfilter.conf'
```

Comment:
Seeing as we've added the overlay into this file should we consider making the filename more generic?

Reply:
Agreed, new commits address this and some surrounding task language.